test/repositories/HistoricEoriRepositorySpec.scala

```diff
@@ -55,6 +56,17 @@ class HistoricEoriRepositorySpec extends SpecBase {
       } yield {})
     }
 
+    "fail to UpdateHistoricEori " in {
+       val mockRepository = mock[DefaultHistoricEoriRepository]
+      when(mockRepository.set(any())).thenReturn(Future.successful(FailedToUpdateHistoricEori))
+
+      await(for {
```

Comment:
Could you please replace with below?

      mockRepository.set(Seq(period1, period2)).map {
        result => result mustBe FailedToUpdateHistoricEori
      }